cmd/containerboot/settings.go

```diff
@@ -195,6 +193,9 @@ func (s *settings) validate() error {
 			return fmt.Errorf("error parsing TS_DEBUG_ADDR_PORT value %q: %w", s.DebugAddrPort, err)
 		}
 	}
+	if s.HealthEnabled && s.HealthCheckAddrPort != "" {
+		return errors.New("TS_HEALTHCHECK_ADDR_PORT is deprecated, use TS_HEALTH_ENABLED in conjunction with TS_LOCAL_ADDR_PORT")
```

Comment:
Could we log in which release this will be removed?

Reply:
We should also note that it's deprecated here https://github.com/tailscale/tailscale/blob/13faa64c142148b1f8c8afd22d61e4a0de651b98/cmd/containerboot/main.go#L55

Reply:
... and I guess we also want to log a warning on line 181

Reply:
Oh yeah that would be better than where I added it. Fixed.